app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  @@renderer = Redcarpet::Markdown.new(Redcarpet::Render::HTML, autolink: true, tables: true)
```

Comment:
Whats the advantage of having the class variable here as opposed to just defining it in the helper method below as a regular variable?